### ISSUE TITLE
Fullgame Features like Progress Saving? Character/Stage Unlocks?

### ISSUE THREAD

**Issue by GenesisMugen** [shinlucho/ikemen-plus#7]
Hi, I like that you're adding more to this, and I'm sure you have a list of things you're trying to port over from Mugen.

I'm just curious if its possible to add some sort of save system to IKEMEN?

**Comment by shinlucho** [comment]
Yup, it's possible to add a unlock feature or even a shop to buy things like MvC2.

On version 0.2 I'll work on saving configurations in option mode and load it from a text file, but I'm not sure if this could work on a unlock system because anyone could edit the text file and unlock whatever they want.

Maybe I need to serialize the progress file or something like that, I need to investigate it.

So yeah, it's possible but doesn't have a high priority atm. Gonna mark this issue as a question, but I'll keep this in mind.
